### PR TITLE
Navigation bar Closes #58

### LINKED ISSUE #58: Nav Bar
Minimal and Simple Nav Bar with white and dark mode toggle button present as well as
home 
editor 
contact 
For inspiration refer [this](https://demos.creative-tim.com/material-kit/index.html?_ga=2.202989457.1035946558.1597326347-1708666551.1597326347)

### PR DESCRIPTION
**Pull request by varunvj1**

### CODE REVIEW

**Comment by varunvj1** [comment]
The branch has no conflicts now. Please review.

File: assets/css/navigation.css [hunk 1]
```diff
@@ -0,0 +1,258 @@
+/* Header section */
+@font-face {
+    font-family: KaushanScript;
+    src: url("../fonts/KaushanScript-Regular.otf") format("opentype");
```
Comment (smaranjitghose) [inline assets/css/navigation.css:4]: Where is this used?

File: assets/css/navigation.css [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+    --button-color: #1496bb;
+
+    --color-sun-moon: linear-gradient(45deg, yellow, red);
+    --glow1: white;
+    --glow2: yellow;
+}
+
+.theme-dark {
+    --color-primary: black;
+    --color-back: rgba(0, 0, 0, 0.8);
```
Comment (smaranjitghose) [inline assets/css/navigation.css:103]: Why again for RGB? When already using Hexcode

File: assets/css/navigation.css (hunk 1 above)
Comment (varunvj1) [reply assets/css/navigation.css:4]: Used this font for "Doc2Pen" in the left.

File: assets/css/navigation.css (hunk 2 above)
Comment (varunvj1) [reply assets/css/navigation.css:103]: I used it to set the precise opacity of the color to 0.8.

**Comment by anushbhatia** [comment]
delete the previous repo. Clone this repo don't use custom css for it. default materialize css with blue text on hover with no bg shadow

**Comment by varunvj1** [comment]
> delete the previous repo. Clone this repo don't use custom css for it. default materialize css with blue text on hover with no bg shadow

So no blue background box effect? Just have to change the color of text to blue by using blue-text?

**Comment by varunvj1** [comment]
I used blue-text class for blue color on active links and removed the background box color. I used as less custom css as possible.

**Comment by anushbhatia** [comment]
make a new pr just add this logo to it nothing else

**Pull request closed without merging**